complete/src/main/java/price/checker/entrypoint/RequestController.java

```diff
@@ -0,0 +1,54 @@
+package price.checker.entrypoint;
+
+import java.io.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.MissingServletRequestParameterException;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import price.checker.domain.StockPrice;
+import price.checker.service.AVRequestService;
+import price.checker.service.StockPriceRpsyService;
+import lombok.extern.slf4j.Slf4j;
+import java.lang.*;
+import java.text.ParseException;
+import java.util.List;
+
+
+/**
+ * This is the controller that passes on user parameters to services to obtain stock prices
+ */
+@RestController
+@Slf4j
+public class RequestController {
+
+    //let Spring do its thing and initialize the two services
+    @Autowired
+    StockPriceRpsyService StockPriceRpsyService;
```

Comment:
keep the bean names lowercase, so: 

    @Autowired
    StockPriceRpsyService stockPriceRpsyService;